### DISCUSSION TITLE
Message is not updated after adding mention during message editing with maximum character count

### DISCUSSION THREAD

**Discussion by VolodLytvynenko** [status-im/status-legacy#16685] [General]
### Steps to reproduce:
1. Send a message with 4095 symbols
2. Open editing composer
3. Enter '@' and mention some user
4. Send the message

### Actual result:
The message is not updated

https://github.com/status-im/status-mobile/assets/52490791/c7c6f606-b70f-4433-bcda-996491e40caf

### Expected result:
The message should be updated with the mention included in the edited message.

### ENV:
Nightly [19 May 2023]

**Comment by OmarBasem** [comment]
I think this is expected since it is already at max character count. Maybe we need to show some alert to the user.

**Comment by ibrkhalil** [comment]
> I think this is expected since it is already at max character count. Maybe we need to show some alert to the user.

Is there validation for composer for exceeding character limit?
@pedro-et @xAlisher
